python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -511,10 +511,12 @@ def run_through_rpc(measure_input, build_result,
         if ref_input:
             args = [nd.array(x, ctx=ctx) for x in ref_input]
         else:
-            # create empty arrays on the remote device and copy them once.
-            # This can avoid some memory issues that make the measurement results unreliable.
+            assert tvm.get_global_func("tvm.contrib.random.random_fill", True), \
+                "Please make sure USE_RANDOM is ON in the config.cmake"
+            random_fill = remote.get_function("tvm.contrib.random.random_fill")
```

Comment:
If the check is on the device. L512 is useless.
In addition, we should use try-catch to capture the error from remote devices and explicitly use "remote" in the error message.

Reply:
Good point. Thanks for pointing this. I will update implementation for rpc.

Reply:
done.